doc/ir-semantics.text

```diff
@@ -187,9 +183,7 @@ Update (partial) function, returns an H:
          Df  = Function f^ (formal*)
          I' := P(Df, lᵥ^)
          l' := l^
-         M  := (materialize P H E osr-map)*
-         H' := H ∪ range(M)
-         E' := (osr-env P H E M osr-map)*
+         (H', E') := sum (create-env H E osr-map)*
```

Comment:
Again, the out-arg needs to be `H+H'` not just `H'`. Would it be useful to use a different metavariable (like I did with M) to make it more apparent that `H'` is just a partial heap?